tests/Persistence/Mapping/AbstractClassMetadataFactoryTest.php

```diff
@@ -72,8 +72,8 @@ public function testItThrowsWhenAttemptingToCheckTransientForShortAlias(): void
         $cmf = $this->getMockForAbstractClass(AbstractClassMetadataFactory::class);
         $this->expectException(MappingException::class);
         /**
-         * @psalm-suppress ArgumentTypeCoercion
-         * @psalm-suppress UndefinedClass
+         * @phpstan-suppress ArgumentTypeCoercion
+         * @phpstan-suppress UndefinedClass
```

Comment:
Suppressing a psalm error with a phpstan-suppress annotation does not make any sense. `@phpstan-suppress` does not exist (the equivalent feature in phpstan uses the `@phpstan-ignore-*` tags, as used just below)